app/lib/src/app.dart

```diff
@@ -4,9 +4,9 @@ import 'package:flutter/services.dart';
 import 'package:hooks_riverpod/hooks_riverpod.dart';
 import 'package:mondaymorning/src/pages/full_error_page.dart';
 import 'package:mondaymorning/src/pages/splash_page.dart';
-import 'package:mondaymorning/src/services/router/mm_router.dart';
 import 'package:mondaymorning/src/services/themes/index.dart';
 import 'package:mondaymorning/src/store/states/app_config/app_config_provider.dart';
+import 'services/router/mm_router.dart';
```

Comment:
```suggestion
import 'package:mondaymorning/src/services/router/mm_router.dart';
```
Avoid using relative imports.